cypress/integration/contextMenu.cy.js

```diff
@@ -0,0 +1,107 @@
+import { AXIS_ID_COLUMNS, AXIS_ID_FILTERS } from '@dhis2/analytics'
+import { TEST_EVENT_DATA } from '../data/index.js'
+import { selectEventProgram } from '../helpers/dimensions.js'
+import {
+    expectAxisToHaveDimension,
+    expectAxisToNotHaveDimension,
+} from '../helpers/layout.js'
+import { goToStartPage } from '../helpers/startScreen.js'
+
+describe('using the main sidebar context menu', () => {
+    const TEST_DIM_ID = 'eventDate'
+    const openContextMenu = () =>
+        cy
+            .getBySel('main-sidebar')
+            .findBySel(`dimension-item-${TEST_DIM_ID}`)
+            .findBySel('dimension-menu-button')
+            .invoke('attr', 'style', 'visibility: initial')
+            .click()
+
+    it('adds item', () => {
+        goToStartPage()
+
+        expectAxisToNotHaveDimension(AXIS_ID_COLUMNS, TEST_DIM_ID)
+        expectAxisToNotHaveDimension(AXIS_ID_FILTERS, TEST_DIM_ID)
+
+        openContextMenu()
```

Comment:
I think the test would read better if you pass in the TEST_DIM_ID (even though the var is available in the fn), so it's easy to understand which item is having its context menu opened.